<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index</class>
 <widget class="QWidget" name="Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>首页</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/shouye.jpg</normaloff>:/images/shouye.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="friend_PB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777214</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>好
友</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>11</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file_PB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>文
件</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>11</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="Friend" name="friendPage"/>
     <widget class="File" name="filePage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Friend</class>
   <extends>QWidget</extends>
   <header location="global">friend.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>File</class>
   <extends>QWidget</extends>
   <header>file.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
